makefiles/libfaketime.mk

```
@@ -0,0 +1,50 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS         += libfaketime
+LIBFAKETIME_VERSION := 0.9.10
+DEB_LIBFAKETIME_V   ?= $(LIBFAKETIME_VERSION)-1
+
+libfaketime-setup: setup
+	$(call GITHUB_ARCHIVE,wolfcw,libfaketime,$(LIBFAKETIME_VERSION),v$(LIBFAKETIME_VERSION))
+	$(call EXTRACT_TAR,libfaketime-$(LIBFAKETIME_VERSION).tar.gz,libfaketime-$(LIBFAKETIME_VERSION),libfaketime)
+	$(call DO_PATCH,libfaketime,libfaketime,-p1)
+
+ifneq ($(wildcard $(BUILD_WORK)/libfaketime/.build_complete),)
+libfaketime:
+	@echo "Using previously built libfaketime."
+else
+libfaketime: libfaketime-setup
+	sed -i "s/LD_PRELOAD/DYLD_INSERT_LIBRARIES/g" $(BUILD_WORK)/libfaketime/man/faketime.1
+	+$(MAKE) -C $(BUILD_WORK)/libfaketime install \
+		PREFIX="$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)" \
+		CC="$(CC) $(CFLAGS)" \
+		LDFLAGS="$(LDFLAGS)" \
+		DESTDIR=$(BUILD_STAGE)/libfaketime
+	$(call AFTER_BUILD)
+endif
+
+libfaketime-package: libfaketime-stage
+	# libfaketime.mk Package Structure
+	rm -rf $(BUILD_DIST)/{,lib}faketime
+	mkdir $(BUILD_DIST)/{,lib}faketime/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)
```

Forgot about this suggestion

```suggestion
	rm -rf $(BUILD_DIST)/{,lib}faketime
	mkdir -p $(BUILD_DIST)/{,lib}faketime/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)
```